stdlib/3/typing.pyi

```diff
@@ -663,3 +663,15 @@ if sys.version_info >= (3, 7):
         def __eq__(self, other: Any) -> bool: ...
         def __hash__(self) -> int: ...
         def __repr__(self) -> str: ...
+
+class _NoneType:
+    __doc__: Optional[str]
+    def __bool__(self) -> bool: ...
+    def __dir__(self) -> Iterable[str]: ...
+    def __format__(self, format_spec: str) -> str: ...
+    def __eq__(self, o: object) -> bool: ...
+    def __ne__(self, o: object) -> bool: ...
+    def __hash__(self) -> int: ...
+    def __repr__(self) -> str: ...
+    def __sizeof__(self) -> int: ...
+    def __str__(self) -> str: ...
```

Comment:
Most of these seem redundant, since they are the same as in `object`. Maybe only `__bool__` is worth including here?